core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,42 +25,48 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <collection-editor-navbar-breadcrumb-directive>
+  </collection-editor-navbar-breadcrumb-directive>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <collection-editor-navbar-directive>
+  </collection-editor-navbar-directive>
+{% endblock local_top_nav_options %}
+
 {% block content %}
   <div ng-controller="CollectionEditor">
-    <div ng-if="collection">
```

Comment:
You removed all of this; this is the main collection editor. This should be moved to the main tab.